lib/device/lib/marionette.js

```diff
@@ -53,7 +53,8 @@ var Marionette = function(device) {
  * @returns {Promise.<MarionetteClient.Client>}
  */
 Marionette.prototype.startSession = function() {
-  debug('[Creating driver] %s:%d connectionTimeout: %d', this.host, this.port, this.timeout);
+  debug('[Creating driver] %s:%d with %dms connection timeout',
```

Comment:
Good call I was thinking that original message should be changed as I made it sound like there was an actual connection timeout and not just a parameter